test/output/basic.p4c.json

```diff
@@ -0,0 +1,558 @@
+{
```

Comment:
Is this the expected output or the generated output? Could we rename things so it is obvious which one it is?